aiozk/connection.py

```diff
@@ -168,27 +172,39 @@ def handle_write(write_future):
             else:
                 f.set_result((zxid, response))
 
+    async def _read(self, size=-1):
+        remaining_size = size
+        payload = b''
+        end_time = time() + self.read_timeout
+        while remaining_size and (time() < end_time):
+            chunk = await self.reader.read(remaining_size)
```

Comment:
Since we have read timeout, maybe would be better not just block here for not definite time, but use `asyncio.wait` with timeout?
Or reader can implement this internally (so `self.reader.read` will take not more than self.read_timeout)